dev-docs/source/DebuggingUnitTests.rst

```diff
@@ -0,0 +1,94 @@
+Debugging Unit Tests
+====================
+
+Debugging typically requires the test executable to be run directly,
+rather than via ctest (which typically spawns off a separate process to
+run the actual tests). So an example of debugging from the command line
+using gdb would be:
+
+| ``   ``\ **$**\ `` gdb bin/AlgorithmsTest``
```

Comment:
The inline literals make this hard to read. How about literal text blocks for this and the other examples on the page?